_locales/en/messages.json

```diff
@@ -203,20 +203,24 @@
         "message": "Settings",
         "description": "Menu item and header for global settings"
     },
-    "off": {
-        "message": "Off",
+    "notificationSettingsDialog": {
+        "message": "When messages arrive, display notifications that reveal:",
+        "description": "Explain the purpose of the notification settings"
+    },
+    "disableNotifications": {
+        "message": "Disable notifications",
         "description": "Label for disabling notifications"
     },
     "nameAndMessage": {
-        "message": "Name and message",
+        "message": "Both sender name and message",
         "description": "Label for setting notifications to display name and message text"
     },
     "noNameOrMessage": {
-        "message": "No name or message",
+        "message": "Only \"You have a new message\"",
```

Comment:
That's not actually what's displayed. The notification is "x New Message(s)"
If you wanna go this route it might be best to use the string name instead of its value.